contracts/mocks/MockVotesOracle.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import '@openzeppelin/contracts/utils/Context.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+
+// MockVotesOracle contract.
+contract MockVotesOracle is Context, Ownable {
```

Comment:
question: Why is this a mock instead of a concrete implementation? 

The Governance needs a real VotesOracle to work, it strike me that even a simple vote manager would be useful (i.e. ensuring access control between this and Governance contract is correct)

Reply:
Well, its kinda is concrete in simplest form. I can rename it to VotesOracle or VotesManager and still use it as a mock, because if we have offchain delegation logic, then we just push voting power snapshot to it as Owner and those functions in the MockVotesOracle are enough for that. If we have some ERC20Votes which allows to getVotes(account, role) and getVotes(account) and has delegations logic in it, then it can be used as oracle for Governance contract.